lib/kubeclient/config.rb

```diff
@@ -63,6 +64,7 @@ def context(context_name = nil)
     private
 
     def ext_file_path(path)
+      return path if Pathname(path).absolute?
```

Comment:
Question: shouldn't this be wrapped with `File` as well? (`return File(path) if Pathname(path).absolute?`